lib/spec.js

```diff
@@ -469,7 +469,7 @@ export default class Spec {
 
     buildRequestBody(additionalOptions, headers, method) {
         let bodyData = {};
-        let requestContentType = headers['content-type'];
+        let requestContentType = headers['content-type'] || this.suite.commonHeaders['content-type'];
```

Comment:
@matmar10 This is not required here, because the headers passed to this function include common headers too.

Reply:
I raised issue #31 because the current behavior does not include common headers set in config. 

If you need, I can create a gist that reproduces this.